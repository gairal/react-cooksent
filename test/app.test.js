import React from 'react';
import renderer from 'react-test-renderer';
import App from '@/app';

describe('App', () => {
  it('Renders correctly default', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
