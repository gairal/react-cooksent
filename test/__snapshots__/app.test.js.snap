// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders correctly default 1`] = `null`;
